ts/torch_handler/base_handler.py

```diff
@@ -180,6 +190,15 @@ def initialize(self, context):
             self.model = setup_ort_session(self.model_pt_path, self.map_location)
             logger.info("Succesfully setup ort session")
 
+        elif self.model_pt_path.endswith(".so") and USE_TORCH_EXPORT:
+            # Set cuda stream to the gpu_id of the backend worker
+            if torch.cuda.is_available() and properties.get("gpu_id") is not None:
+                torch.cuda.set_stream(torch.cuda.Stream(int(properties.get("gpu_id"))))
```

Comment:
could you add some comment here? Why are we launching things on specific streams?